dist/formats/financial_releases_campaign/frontend/schema.json

```diff
@@ -196,6 +196,9 @@
         },
         "footer_legal_disclaimer": {
           "type": "string"
+        },
+        "pdf": {
+          "$ref": "#/definitions/image"
```

Comment:
Not sure this is the correct definition.
https://github.com/alphagov/govuk-content-schemas/blob/03e412bbbc5cdd22a2a6c235b51df2518bbcfc56/formats/travel_advice/publisher/details.json#L80 defines an `asset_link`, it might make more sense to move this definition into `formats/definitions.json` and re-use it.